### DISCUSSION TITLE
Wipe archive, error 403, "message":"Missing Access","code":50001

### DISCUSSION THREAD

**Discussion by Kamatii** [victornpb/undiscord#613] [1. Questions & Answers]
Heyo,

I'm not a coder, but I think I know what this means.

Issue: I got my json data (archive) from discord and went ahead to try wiping it with discrub.
Issue is, I have left many servers and with them, left some messages in.
I know it's not possible to remove messages from a server I no longer have access to, that is perfectly fine for me.

But, the archive wipe throws the 403 error, then stops, doesn't do anything...
I don't care if it can't access certain messages to remove, I want it to skip if it can't access the channel or server, and continue with the next.

Manually editing the index.json is out of question, the sheer number of servers I no longer have access to would take me days to manually remove myself.

So in short, how do I make the script throw a console error for 403 code 50001, but continue with the next server instead of stopping?
